packages/math-input/src/data/keys.ts

```diff
@@ -108,6 +108,12 @@ type Key =
     | "W"
     | "X"
     | "Y"
-    | "Z";
+    | "Z"
+
+    // Perseus' Expression MathInput
```

Comment:
I don't follow what this means. Are these special keys specific to the Expression input?

Reply:
Yeah, that's right. I'll update the comment to explain better. We'll probably want to look at whether we need these anymore; it's hard to imagine that they're important if they're only in the desktop experience.